src/wallet/scriptpubkeyman.cpp

```diff
@@ -1385,9 +1385,10 @@ void LegacyScriptPubKeyMan::ReturnDestination(int64_t nIndex, bool fInternal, co
     WalletLogPrintf("keypool return %d\n", nIndex);
 }
 
-bool LegacyScriptPubKeyMan::GetKeyFromPool(CPubKey& result, const OutputType type, bool internal)
+bool LegacyScriptPubKeyMan::GetKeyFromPool(CPubKey& result, const OutputType type)
 {
     assert(type != OutputType::BECH32M);
+    const bool internal{};
     if (!CanGetAddresses(internal)) {
```

Comment:
The following style is more consistent with what we do in other places:
'''cpp
if (!CanGetAddresses(/*internal*/=false)) {
``

Reply:
There's three uses of internal so defining a local bool seemed cleaner. Update all three locations in the style you linked or leave it as it is?

Reply:
Updated.